sencha-workspace/SlateAdmin/app/model/validator/CallbackValidator.js

```diff
@@ -0,0 +1,34 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+/**
+ * Validates that the passed value is not `null` or `undefined` or `''`.
+ */
+Ext.define('SlateAdmin.model.validator.CallbackValidator', {
```

Comment:
@BillClinton let's move this up to just `SlateAdmin.validator.CallbackValidator`